cni/cmd/tc-redirect-tap/main.go

```diff
@@ -229,9 +274,105 @@ func (p plugin) add(currentResult *current.Result) error {
 }
 
 func (p plugin) del() error {
-	panic("del is currently unimplemented")
+	return p.netNS.Do(func(_ ns.NetNS) error {
+		var multiErr *multierror.Error
+
+		// try to remove the qdisc we added from the redirect interface
+		redirectLink, err := p.GetLink(p.redirectInterfaceName)
+		switch err.(type) {
+
```

Comment:
nit: I think we can get rid of the blank lines before the first `case`.

Reply:
I put it there because I think it's easier to read/navigate when each case is a separate block. If there's disagreement I can change it.

Reply:
I'm not a fan, but it's not a big deal.  Feel free to leave it.